packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import { useRef } from 'react';
```

Comment:
Can we import this from `@wordpress/element`?

Reply:
Yes, good idea!